src/commonMain/kotlin/org/kbson/internal/Base64Utils.kt

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kbson.internal
+
+import kotlin.experimental.and
+
+/** Base64 utils to convert to and from a Base64 encoded string and ByteArray */
+@Suppress("MagicNumber")
+public object Base64Utils {
```

Comment:
Can all these public classes in `org.kbson.internal` be given the `internal` modifier instead `public`, and if not, why not?

Reply:
Done